hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -77,6 +77,7 @@
 
 	private static final String PROTO_FILE_EXTENSION = ".pb";
 	private static final String CSV_FILE_EXTENSION = ".csv";
+	private static final int DEFAULT_EXPORT_PERIOD = 600; // seconds
```

Comment:
Note that all global/dynamic properties **must** have a default value from the JAR's _src/main/resources/bootstrap.properties_. So we never have to use inline default constants in the code. 👍

Reply:
Removed this line. It's good to know all properties must have a default value. Thx.